titration/ui_state/demo_mode/demo_temp_probe.py

```diff
@@ -54,37 +46,47 @@ def loop(self):
         The function to loop through and display to the LCD screen until a new keypad input
         """
         if self.substate == 1:
-            self.titrator.lcd.print("1: Get Temperature", line=1)
-            self.titrator.lcd.print("2: Get Resistance", line=2)
-            self.titrator.lcd.print("", line=3)
-            self.titrator.lcd.print("4: Return", line=4)
+            self.titrator.lcd.print("1: Get Temp One", line=1)
+            self.titrator.lcd.print("2: Get Res One", line=2)
+            self.titrator.lcd.print("3: Get Temp Two", line=3)
+            self.titrator.lcd.print("4: Get Res Two", line=4)
 
         elif self.substate == 2:
-            self.titrator.lcd.print("Probe Temperature", line=1)
+            self.titrator.lcd.print("Probe One Temp", line=1)
             self.titrator.lcd.print(
-                f"{self.titrator.temp_probe.get_temperature()} C",
+                f"{self.titrator.temp_probe_one.get_temperature()} C",
                 line=2,
                 style="center",
             )
             self.titrator.lcd.print("Press any to cont.", line=3)
             self.titrator.lcd.print("", line=4)
 
         elif self.substate == 3:
-            self.titrator.lcd.print("Probe Resistance", line=1)
+            self.titrator.lcd.print("Probe One Res", line=1)
             self.titrator.lcd.print(
-                f"{self.titrator.temp_probe.get_resistance()} Ohms",
+                f"{self.titrator.temp_probe_one.get_resistance()} Ohms",
                 line=2,
                 style="center",
             )
             self.titrator.lcd.print("Press any to cont.", line=3)
             self.titrator.lcd.print("", line=4)
 
         elif self.substate == 4:
-            self.titrator.lcd.print("Temperature Control", line=1)
-            self.titrator.lcd.print("Activated", line=2, style="center")
+            self.titrator.lcd.print("Probe Two Temp", line=1)
             self.titrator.lcd.print(
-                f"{self.titrator.temp_probe.get_temperature()} C", line=3
+                f"{self.titrator.temp_probe_two.get_temperature()} C",
+                line=2,
+                style="center",
             )
-            self.titrator.lcd.print("Press any to stop", line=4)
+            self.titrator.lcd.print("Press any to cont.", line=3)
+            self.titrator.lcd.print("", line=4)
 
-            self.titrator.temp_controller.update()
+        elif self.substate == 5:
+            self.titrator.lcd.print("Probe Two Res", line=1)
+            self.titrator.lcd.print(
+                f"{self.titrator.temp_probe_two.get_resistance()} Ohms",
+                line=2,
+                style="center",
+            )
+            self.titrator.lcd.print("Press any to cont.", line=3)
```

Comment:
Does this text fit? "Any key to continue" This way you don't need an abbreviation.

Reply:
There are many UI_States that use the phrase "Press any to cont.", would you like me to just create a separate UI PR that addresses all the UI States that uses this wording?